database/java/2018/1000100.yaml

```diff
@@ -0,0 +1,34 @@
+---
+cve: 2018-1000100
+title: CVE in org.nd4j:nd4j-buffer
+description: >
+    GPAC MP4Box version 0.7.1 and earlier contains a Buffer Overflow vulnerability in src/isomedia/avc_ext.c lines 2417 to 2420 that can result in Heap chunks being modified, this could lead to RCE. This attack appear to be exploitable via an attacker supplied MP4 file that when run by the victim may result in RCE.
```

Comment:
The title and description belong to 2 different projects I presume. The vulnerability is for gpac while the title says it is for package org.nd4j:nd4j-buffer. 
@msrb. PTAL

Reply:
Also, I think fix is available in the nightly builds, but and official version hasn't been released yet.
https://gpac.wp.imt.fr/downloads/gpac-nightly-builds/ .

Reply:
This looks like a FP. See the github repo in references: https://github.com/gpac/gpac

Not a Java project :)

Reply:
@msrb oops. Closing this one